compose/wordpress.md

```diff
@@ -130,12 +130,14 @@ browser.
 
 ### Shutdown and cleanup
 
-The command [docker-compose down](/compose/reference/down.md) removes the
+The command `docker-compose down` removes the
 containers and default network, but preserves your WordPress database.
 
 The command `docker-compose down --volumes` removes the containers, default
 network, and the WordPress database.
 
+[Learn more about the `docker-compose down` command](/compose/reference/down.md)
```

Comment:
See my other comment; I think for this page it may be good to keep these inline, so that they don't distract from the example itself.